src/Models/RaspberryPiDevice.ts

```diff
@@ -42,6 +42,32 @@ export class RaspberryPiDevice extends ContainerDeviceBase {
         DeviceType.Raspberry_Pi, templateFilesInfo);
   }
 
+  private async getBinaryFileName(): Promise<string|undefined> {
+    // Parse binary name from CMakeLists.txt file
+    const cmakeFilePath = path.join(this.projectFolder, 'CMakeLists.txt');

```

Comment:
There is a constant definition for “CMakeLists.txt” which you can leverage.

Reply:
The constant is in `DigitalTwinConstants.ts`. It seems not good to use constant in that scope.

Reply:
Then move it to a generic place.